autotest/test_grid.py

```diff
@@ -379,12 +379,8 @@ def test_unstructured_xyz_intersect(example_data_path):
     ncpl = np.array(3 * [len(iverts)])
     nnodes = np.sum(ncpl)
 
-    top = np.ones(
-        (nnodes),
-    )
-    botm = np.ones(
-        (nnodes),
-    )
+    top = np.ones((nnodes))
+    botm = np.ones((nnodes))
```

Comment:
This probably should be corrected to `np.ones((nnodes,))` (or simply `np.ones(nnodes)`)